server/weaverbird/backends/sql_translator/steps/fillna.py

```diff
@@ -0,0 +1,40 @@
+from weaverbird.backends.sql_translator.steps.utils.query_transformation import (
+    build_selection_query,
+)
+from weaverbird.backends.sql_translator.types import (
+    SQLPipelineTranslator,
+    SQLQuery,
+    SQLQueryDescriber,
+    SQLQueryExecutor,
+    SQLQueryRetriever,
+)
+from weaverbird.pipeline.steps import FillnaStep
+
+
+def translate_fillna(
+    step: FillnaStep,
+    query: SQLQuery,
+    index: int,
+    sql_query_retriever: SQLQueryRetriever = None,
+    sql_query_describer: SQLQueryDescriber = None,
+    sql_query_executor: SQLQueryExecutor = None,
+    sql_translate_pipeline: SQLPipelineTranslator = None,
+    subcall_from_other_pipeline_count: int = None,
+) -> SQLQuery:
+    query_name = f'FILLNA_STEP_{index}'
+
+    unchanged_colums = query.metadata_manager.retrieve_query_metadata_columns_as_str(
+        columns_filter=step.columns
+    )
+    fill_value = f"'{step.value}'" if isinstance(step.value, str) else step.value
```

Comment:
This can be problematic if we have in `step.value` a single quotes for the final query...
I don't know... but it could be a good idea todo some replace here... just to be sure ?
```python
# we escape weird quotes here
step.value = step.value.replace('"', "'").replace("'", "\\'")

fill_value = f"'{step.value}'" if isinstance(step.value, str) else step.value
```

if this rmq is relevant... i thing we need to add tests for that behaviour too ? 🤔

Reply:
:+1:  good point. 
I'll add an util for that.